README.md

```diff
@@ -1 +1,59 @@
-# jobs-Nada-agha
\ No newline at end of file
+# jobs-Nada-agha
+
+### Check out the Live Link : [website](https://test-n-a.herokuapp.com/)
+
+---
+
+## Description:
+
+-  This Web app for searching about the jops and get the all information about it.
+
+---
+
+## User Journey:
+
+- When a user enters the website they will see a search bar.
+- They could search for a specific jop.
+- The user can click on the search jop to show the details.
+
+---
+
+## User Stories:
+
+- [x] As a user, I want to be able to see the search bar.
+- [x] As a user, I want to be able to search about specific jop.
+- [x] As a user, I want to be able to select jop.
+- [x] As a user, I want to be able to see the details.
+
+## Installation
```

Comment:
nice readme, but the installation guide needs more details like `install dependencies using npm i` `create .env file in the root ...` and `run the dev server using npm run dev` ...etc